_docs/backenders/strategy-2021/roles/capacity.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Capacity
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Capacity role definition
+
+**Current responsible**: Alicia A. Arenzana
+
+## Responsibility:
+    **Meetings and other FA:**
+- Facilitator on the capacity part of the BE SMO meeting.
+- Attend to capacity monthly meeting.
+- Participate in BD/FE retrospectives every 6 months.
+- Don't leave unanswered other functional areas requests for occasional support on small tasks (i.e.: DNS changes, restart servers, legacy project related work, etc).
```

Comment:
I'd frame this in a more positive way, like: Answer other functional area requests for occasional support...